aws-android-sdk-cognitoidentityprovider/src/main/java/com/amazonaws/mobileconnectors/cognitoidentityprovider/CognitoUser.java

```diff
@@ -474,8 +509,10 @@ public void resendConfirmationCode(final VerificationHandler callback) {
 
     /**
      * Internal method to request registration code resend.
+     * @param clientMetadata A map of custom key-value pairs that is passed to the lambda function for
+     *                       custom workflow.
      */
-    private ResendConfirmationCodeResult resendConfirmationCodeInternal() {
+    private ResendConfirmationCodeResult resendConfirmationCodeInternal(final Map<String, String> clientMetadata) {
```

Comment:
where is this `clientMetadata` being used in this function?

Reply:
Good catch - I needed to add it to the ResendConfirmationCodeRequest - fixed.